.github/actions/go-tools-cache/action.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Load Go tools cache'
+description: 'Load Go tools cache'
+runs:
+  using: "composite"
+  steps:
+    - name: Use GNU tar instead BSD tar
+      if: runner.os == 'Windows'
+      shell: bash
+      run: echo 'C:\Program Files\Git\usr\bin' >> $GITHUB_PATH
+
+    - name: Load Go tools cache
+      id: go-tools-cache
+      uses: actions/cache@v4
+      with:
+        path: |
+          ~/go/pkg
+          ~\go\pkg
+          ~/go/bin
+          ~\go\bin
+        key: ${{ runner.os }}-go-${{ env.GO_VERSION }}-v6-tools-${{ hashFiles('**/go.sum') }}
```

Comment:
`bin` is enought here, right?

Reply:
According to the documentation I have read it should be sufficient. https://github.com/actions/cache/blob/main/examples.md#go---modules . I suspect that `bin` is generated next to the `pkg` folder.

Reply:
:thinking: tools are some compiled binaries we use, generic cache documentation is not relevant here, right?